RxTodo/View/ViewControllers/TodoTableView/TodoViewController.swift

```diff
@@ -11,39 +11,45 @@ import RxSwift
-            Todo.init(title: "Third", content: "Lorem ipsum dolor sit er elit lamet, consectetaur cillium adipisicing pecu, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. Nam liber te conscient to factor tum poen legum odioque civiuda.", tags: ["Third", "dev"])]
-
+             Todo.init(title: "Second", content: "Lorem ipsum dolor sit er elit lamet, consectetaur cillium adipisicing pecu, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. Nam liber te conscient to factor tum poen legum odioque civiuda.", tags: ["Home", "wiki"]),
+             Todo.init(title: "Third", content: "Lorem ipsum dolor sit er elit lamet, consectetaur cillium adipisicing pecu, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. Nam liber te conscient to factor tum poen legum odioque civiuda.", tags: ["Third", "dev"])]
+        
         self.todoModel.setTodoModelTemo(todoPool)
         
         //MARK: ####### END ######
         
-        self.todoModel.todoModelObservable.bind(to: tableView.rx.items(cellIdentifier: "TodoTableViewCell", cellType: TodoTableViewCell.self)) {
-            row, todoItem, cell in
-            cell.titleLabel.text = todoItem.title
-            cell.contentLabel.text = todoItem.content
-            cell.tagsLabel.text = String.stringOfTagsFromArray(todoItem.tags)
-        }.disposed(by: disposeBag)
-
-
-        tableView.rx.modelSelected(Todo.self)
-            .subscribe(onNext: { [weak self] (todoItem) in
+        self.todoModel
+            .todoModelObservable
```

Comment:
IMO, better to name it as:
```suggestion
            .asObservable
```